tech/languages/java/java-build-tools-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Java Build Tools
+page: java
+---
+
+## Maven installation
+
+Maven is probably the most popular build tool in Java world these days. It's available in official Fedora repositories, in form of RPM. To install it, simply type:
+
+```
+# dnf install maven
+```
```

Comment:
Ok, let's forget that for now. But please mention an example of using the build tools. Just a simple one.

p.s. just wondering, is there an app in fedora that uses packaged version of Spring framework?